tests/datasource/fluent/_fake_cloud_api.py

```diff
@@ -470,9 +468,9 @@ def get_expectation_suites_cb(request: PreparedRequest) -> CallbackResult:
     queried_names: Sequence[str] = query_params.get("name", [])  # type: ignore[assignment]
 
     exp_suites: dict[str, dict] = _CLOUD_API_FAKE_DB["expectation_suites"]
-    exp_suite_list: list[dict] = list(exp_suites.values())
+    exp_suite_list: list[dict] = [d["data"] for d in exp_suites.values()]
```

Comment:
we shouldn't be putting `data` in our fake db, but I don't want to do the more holistic update now. This change just prevents us from getting a `data` array with nested `data` objects.